nginx/etc/nginx/nginx.conf

```diff
@@ -21,7 +21,12 @@ http {
 
     access_log /var/log/nginx/access.log timed_combined;
 
-    keepalive_timeout 65;
+    keepalive_disable msie6;
+    keepalive_timeout 65s;
+    keepalive_requests 100;
```

Comment:
This defaults to 100 and might actually need to be higher, but I don't have a good sense for it yet.